src/GafferBindings/CompoundNumericPlugBinding.cpp

```diff
@@ -54,6 +56,23 @@ namespace
 {
 
 template<typename T>
+std::string maskedCompoundNumericPlugRepr( const T *plug, unsigned flagsMask )
+{
+	std::string extraArgs = "";
+
+	const IECore::GeometricData::Interpretation interpretation = plug->interpretation();
+	if( interpretation != IECore::GeometricData::None && interpretation != IECore::GeometricData::Color )
```

Comment:
Why wouldn't we preserve `GeometricData::Color`? It'd be an odd choice for someone to make perhaps, but I don't see why we'd want the serialisation to silently throw it away.